### PR TITLE
Add github pages version of the document

### PR DESCRIPTION
**Pull request by relekang**
Published with `make publish`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,12 +11,27 @@ test: compile
...
+jekyll: clean gh-pages/index.html gh-pages/statutter.pdf
+
+publish: clean gh-pages/index.html gh-pages/statutter.pdf
+	rm -rf gh-pages/_site
+	ghp-import gh-pages
+	git push origin gh-pages -f
+
+gh-pages/index.html:
+	echo "---\nlayout: default\n---" > gh-pages/index.html
+	pandoc -f latex -t html content.tex >> gh-pages/index.html
```
Comment (ekmartin) [inline Makefile]: Do you want to move everything to a `content.tex` file or is this supposed to be `main.tex`?

**Comment by relekang** [comment]
Content.tex is imported into main.tex because the html conversion needs a document with the content only

File: Makefile (hunk 1 above)
Comment (relekang) [inline Makefile]: Content.tex is imported into main.tex because the html conversion needs a document with the content only

Comment (ekmartin) [inline Makefile]: I see :)

**Comment by ekmartin** [comment]
:shipit:

**Pull request merged**